cmd/registration-operator/main.go

```diff
@@ -47,6 +47,8 @@ func newNucleusCommand() *cobra.Command {
 	}
 
 	cmd.AddCommand(hub.NewHubOperatorCmd())
+	cmd.AddCommand(hub.NewHubManagerCmd())
+	cmd.AddCommand(hub.NewWebhookCmd())
```

Comment:
What commands are these replacing?

Reply:
Oh gotcha. It would be helpful to have some comments or command descriptions that explain that. As a newcomer to the project, it is often difficult to tell which command/manager is which for spoke singleton vs. separate.

Reply:
yeah, that is a good suggestion.